client/a8c-for-agencies/sections/migrations/migrations-overview/index.tsx

```diff
@@ -1,26 +1,155 @@
+import { Button, Card, WordPressLogo } from '@automattic/components';
+import { Icon, external } from '@wordpress/icons';
 import { useTranslate } from 'i18n-calypso';
+import { useCallback } from 'react';
 import Layout from 'calypso/a8c-for-agencies/components/layout';
 import LayoutBody from 'calypso/a8c-for-agencies/components/layout/body';
 import LayoutHeader, {
 	LayoutHeaderTitle as Title,
+	LayoutHeaderActions as Actions,
 } from 'calypso/a8c-for-agencies/components/layout/header';
 import LayoutTop from 'calypso/a8c-for-agencies/components/layout/top';
 import MobileSidebarNavigation from 'calypso/a8c-for-agencies/components/sidebar/mobile-sidebar-navigation';
+import { A4A_REFERRALS_BANK_DETAILS_LINK } from 'calypso/a8c-for-agencies/components/sidebar-menu/lib/constants';
+import StatusBadge from 'calypso/a8c-for-agencies/sections/referrals/common/step-section-item/status-badge';
+import useGetTipaltiPayee from 'calypso/a8c-for-agencies/sections/referrals/hooks/use-get-tipalti-payee';
+import { getAccountStatus } from 'calypso/a8c-for-agencies/sections/referrals/lib/get-account-status';
+import tipaltiLogo from 'calypso/a8c-for-agencies/sections/referrals/lib/tipalti-logo';
+import pressableIcon from 'calypso/assets/images/pressable/pressable-icon.svg';
+import { useDispatch } from 'calypso/state';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+
+import './style.scss';
 
 export default function MigrationsOverview() {
 	const translate = useTranslate();
+	const dispatch = useDispatch();
 	const title = translate( 'Migrations' );
 
+	const { data } = useGetTipaltiPayee();
+	const accountStatus = getAccountStatus( data, translate );
+	const statusProps = {
+		children: accountStatus?.status,
+		type: accountStatus?.statusType,
+		tooltip: accountStatus?.statusReason,
+	};
+
+	const onAddBankDetailsClick = useCallback( () => {
+		dispatch( recordTracksEvent( 'calypso_a4a_referrals_add_bank_details_button_click' ) );
```

Comment:
Can we keep the event name different for this?

```suggestion
		dispatch( recordTracksEvent( 'calypso_a4a_migrations_add_bank_details_button_click' ) );
```